index.html

```diff
@@ -1,25 +1,31 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <link rel="stylesheet" href="style.css">
+        <meta charset="UTF-8" />
```

Comment:
These formatting changes were made automatically by [Prettier](https://prettier.io/), another tool for enforcing consistent styling.

The simplest way to use is to install [the vscode plugin](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) and enable formatting on save.